### ISSUE TITLE
Add a suffix element inside radio and checkbox component

### ISSUE THREAD

**Issue by marineFabien** [adeo/mozaic-design-system#665]
# 🚀 Styleguide feature request

### Description
Regarding `Mozaic-freemarker` we need to implement suffix span inside **radio** and **checkbox** label component in order to be clean about architecture of this project. We had disscussed about that before with Trésor.

You can see this here: 
https://leroymerlin-team-core-dev-q4yba2-uat1.nprd-02-a9ef.priv.manawa.adeo.cloud/sommaire-team-core/formulaire

Basically it is only a `span` inside a `label` as is below : 
```
<label aria-hidden="true" class="mc-checkbox__label" for="mc-checkbox-id">My label
     <span class="mc-checkbox__suffix-label>My suffix</span>
</label>
```
with its style : 
```
.mc-checkbox__suffix-label {
   color: $color-grey-500;
   cursor: pointer;
}
```

Thank you :)

**Comment by caqueste** [comment]
@tiloyi @mohamedMok To add to MDS in a future sprint + Suffix need to use the `Grey-600` colour.

**Comment by tiloyi** [comment]
The ball is in Pauline Lapierre's court. 
We are waiting for her to come back, to see how this component should evolve.

**Comment by tiloyi** [comment]
Hi @caqueste,
Is this planned for V2 or should we close this issue?

**Comment by caqueste** [comment]
@tiloyi You can close it ;)
